devel/jam/Portfile

```diff
@@ -24,7 +24,15 @@ long_description    Jam is a software build tool (like make) from \
 homepage            http://www.perforce.com/jam/jam.html
 master_sites        ftp://ftp.perforce.com/jam/
 
-distfiles           ${name}-${version}.tar
+# use_tar is new in MacPorts 2.5.0.
+if {[info command use_tar] eq "use_tar"} {
```

Comment:
We have a [utility function](https://github.com/macports/macports-base/blob/a107e0952259d1e2cb202b22e4f156f051865d39/src/port1.0/portutil.tcl#L75) for checking for the existence of an option. This should read:
```
if {[exists use_tar]} {
...
}
```

Reply:
I copied this `[info command ...]` idiom out of the deco-archive port, where it's used to verify the existence of the `macports_version` proc before calling it. I note that `use_tar` and similar features aren't declared as options.

Reply:
Hm, you are right. In fact, `use_*` are options, but `exists` only works with a default value. I would consider this a bug in base.

Reply:
Er, yes, that's what I meant. They're options, but they don't set their defaults. I was surprised to learn that we have options that don't set defaults.